.gitignore

```diff
@@ -39,3 +39,6 @@ dist/
 .turbo/
 packages/og-image/vender/*.wasm
 .wrangler/
+
+.git
+.gitignore
```

Comment:
Why did you add this? 🤔

![CleanShot 2024-01-17 at 14 42 35@2x](https://github.com/graphprotocol/docs/assets/1059139/f8776af4-421d-4161-8e59-6b45cdd036ca)

Reply:
Ah ha, thank you!